b2sdk/bucket.py

```diff
@@ -604,8 +604,9 @@ def _find_unfinished_file_if_possible(self, upload_source, file_name, file_info,
 
         This is only possible if the application key being used allows ``listFiles`` access.
         """
+        file_prefix = file_name[0:file_name.rindex('/')] if '/' in file_name else file_name
```

Comment:
```suggestion
        file_prefix = file_name
```
I'm pretty sure this should work (and more efficiently)

Reply:
Fixed